src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -179,13 +179,13 @@ public static class Font
 			/// </summary>
 			public static bool IgnoreTextScaleFactor { get; set; } = false;
 
-#if __IOS__
+#if __ANDROID__ || __IOS__ 
 			/// <summary>
 			/// Allows the user to limit the scale factor without having to ignore it.
 			/// </summary>
-			public static double MaximumTextScaleFactor { get; set; } = 2.0;
+			public static double? MaximumTextScaleFactor { get; set; }
```

Comment:
just type this as `float`, since the consumers are all expecting float anyways

Reply:
should I keep the ?

Reply:
Yes, keep the nullable, so the default does not change (no maximum).